packages/node-http-handler/src/node-http-handler.spec.ts

```diff
@@ -18,12 +18,12 @@ import {
 describe("NodeHttpHandler", () => {
   describe("constructor", () => {
     it("can set httpAgent and httpsAgent", () => {
-      let maxSockets = Math.round(Math.random() * 50);
+      let maxSockets = Math.round(Math.random() * 50) + 1;
       let nodeHttpHandler = new NodeHttpHandler({
         httpAgent: new http.Agent({ maxSockets }),
       });
       expect((nodeHttpHandler as any).httpAgent.maxSockets).toEqual(maxSockets);
-      maxSockets = Math.round(Math.random() * 50);
+      maxSockets = Math.round(Math.random() * 50) + 1;
```

Comment:
```suggestion
      maxSockets = Math.ceil(Math.random() * 50);
```

Reply:
Doesn't work as `Math.random()` is inclusive of 0 from https://github.com/aws/aws-sdk-js-v3/pull/1660#discussion_r518284405